numpy/lib/_function_base_impl.py

```diff
@@ -4400,139 +4267,82 @@ def quantile(a,
 
     Notes
     -----
-    In general, the quantile at probability level :math:`q` of a cumulative
-    distribution function :math:`F(y)=P(Y \\leq y)` with probability measure
-    :math:`P` is defined as any number :math:`x` that fulfills the
-    *coverage conditions*
+    Given a sample `a` from an underlying distribution, `quantile` provides a
```

Comment:
Sometimes it's `` `a` `` sometimes its ``` ``a`` ``` and it renders differently.

Reply:
According to numpydoc standards , single backticks are correct here because `a` is an argument of this function. I see that you added some double backticks in gh-24254 that I didn't notice, since I wrote this before that PR merged. I'll fix that.